mods/ts/rules/defaults.yaml

```diff
@@ -123,6 +124,114 @@
 	ExternalCondition@CRATE-CLOAK:
 		Condition: crate-cloak
 
+^1x2Shape:
```

Comment:
I wanted to match the Building.Footprint, but I guess it would be more intuitive to change the LocalYaw rotation into the other direction and switch these around like you suggested.